core/src/test/java/com/github/dozermapper/core/util/ReflectionUtilsTest.java

```diff
@@ -101,7 +101,7 @@ public void testGetInterfacePropertyDescriptors() {
         assertEquals(1, descriptors.length);
 
         descriptors = ReflectionUtils.getInterfacePropertyDescriptors(TestClass.class);
-        assertEquals(4, descriptors.length);
+        assertEquals(3, descriptors.length);
```

Comment:
Whys the test changed? If its just a dep upgrade, i'd not expect this.

Reply:
Good question - the test fails unless I reduce the count to 3.

Reply:
have you had chance to investigate this? i cant really accept the merge unless we know why the test needs to be changed...

Reply:
This is part of the CVE fix in beanutils 1.9.4 that prohibits accessing the `class` property descriptor by default now, see `Jira520TestCase` in [https://github.com/apache/commons-beanutils/pull/7/commits/2780a77600e6428b730e3a5197b7c5baf1c4cca0](https://github.com/apache/commons-beanutils/pull/7/commits/2780a77600e6428b730e3a5197b7c5baf1c4cca0)

beanutils 1.9.3:

```
java.beans.PropertyDescriptor[name=c; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@69ee81fc; required=false}; propertyType=class java.lang.String; readMethod=public java.lang.String com.github.dozermapper.core.util.DozerTest$TestClass.getC()]
java.beans.PropertyDescriptor[name=class; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@6e2aa843; required=false}; propertyType=class java.lang.Class; readMethod=public final native java.lang.Class java.lang.Object.getClass()]
java.beans.PropertyDescriptor[name=a; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@6f36c2f0; required=false}; propertyType=class java.lang.String; readMethod=public abstract java.lang.String com.github.dozermapper.core.util.DozerTest$TestIF1.getA(); writeMethod=public abstract void com.github.dozermapper.core.util.DozerTest$TestIF1.setA(java.lang.String)]
java.beans.PropertyDescriptor[name=b; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@f58853c; required=false}; propertyType=class java.lang.Integer; readMethod=public abstract java.lang.Integer com.github.dozermapper.core.util.DozerTest$TestIF2.getB()]
```

beanutils 1.9.4:

```
java.beans.PropertyDescriptor[name=c; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@376a0d86; required=false}; propertyType=class java.lang.String; readMethod=public java.lang.String com.github.dozermapper.core.util.DozerTest$TestClass.getC()]
java.beans.PropertyDescriptor[name=a; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@62656be4; required=false}; propertyType=class java.lang.String; readMethod=public abstract java.lang.String com.github.dozermapper.core.util.DozerTest$TestIF1.getA(); writeMethod=public abstract void com.github.dozermapper.core.util.DozerTest$TestIF1.setA(java.lang.String)]
java.beans.PropertyDescriptor[name=b; values={expert=false; visualUpdate=false; hidden=false; enumerationValues=[Ljava.lang.Object;@17d919b6; required=false}; propertyType=class java.lang.Integer; readMethod=public abstract java.lang.Integer com.github.dozermapper.core.util.DozerTest$TestIF2.getB()]
```

I'm not sure if dozer really needs the `class` property descriptor for anything, though.

Reply:
thanks for update. sounds fine by me.